app/src/main/java/com/iic/shopingo/ui/activities/RequestList.java

```diff
@@ -0,0 +1,140 @@
+package com.iic.shopingo.ui.activities;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.BaseAdapter;
+import android.widget.ListView;
+import com.iic.shopingo.data.Request;
+import com.iic.shopingo.ui.RequestListItem;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by asafg on 03/03/15.
+ */
+public class RequestList extends Activity implements AdapterView.OnItemClickListener {
+
+  private ListView list;
+  private RequestListAdapter adapter;
+  private OnRequestHandledListener listener;
+
+  public interface OnRequestHandledListener {
+    public void onRequestAccepted(Request request);
+    public void onRequestRejected(Request request);
```

Comment:
@AsafGartner @agelber 
Lets be consistent with the terminology, we currently have accepted, approved, rejected, declined.
Don't care which but I think we should stick with Accepted/Declined or Approved/Rejected.